### PR TITLE
Disable upsell in signup while creating account.

### PR DESCRIPTION
**Pull request by vincaslt**
During payment or verification, upsell should not be available.

### CODE REVIEW

File: src/app/containers/SignupContainer/SignupContainer.js [hunk 1]
```diff
@@ -167,7 +173,7 @@ const SignupContainer = ({ history, onLogin }) => {
                             <PaymentStep
                                 model={model}
                                 paymentAmount={selectedPlan.price.total}
-                                onPaymentDone={handlePaymentDone}
+                                onPay={(...rest) => withLoading(handlePayment(...rest))}
```
Comment (EpokK) [inline src/app/containers/SignupContainer/SignupContainer.js]: Do you really need `rest` here?
```suggestion
                                onPay={(data) => withLoading(handlePayment(data))}
```

File: src/app/containers/SignupContainer/SignupContainer.js [hunk 2]
```diff
@@ -157,7 +162,8 @@ const SignupContainer = ({ history, onLogin }) => {
                             <VerificationStep
                                 model={model}
                                 allowedMethods={signupAvailability.allowedMethods}
-                                onVerificationDone={handleVerificationDone}
+                                onVerify={(...rest) => withLoading(handleVerification(...rest))}
```
Comment (EpokK) [inline src/app/containers/SignupContainer/SignupContainer.js]:
```suggestion
                                onVerify={(data) => withLoading(handleVerification(data))}
```

Comment (vincaslt) [reply src/app/containers/SignupContainer/SignupContainer.js]: handle payment done requires two parameters: `(model, paymentParameters)` `...rest` seems like a good way to tell that this method doesn't care about params.

File: src/app/containers/SignupContainer/SignupContainer.js (hunk 1 above)
Comment (vincaslt) [reply src/app/containers/SignupContainer/SignupContainer.js]: handle payment done requires two parameters: (model, paymentParameters) ...rest seems like a good way to tell that this method doesn't care about params.

File: src/app/containers/SignupContainer/SignupContainer.js (hunk 2 above)
Comment (vincaslt) [reply src/app/containers/SignupContainer/SignupContainer.js]: same with handle verifications: `(model, code, params)`

**Pull request merged**